roblox/badges.py

```diff
@@ -44,16 +44,16 @@ class Badge(BaseBadge):
     Represents a badge from the API.
 
     Attributes:
-        id: The badge Id.
```

Comment:
is it "a badge ID" or "a badge's ID"? i honestly don't know what's best, but considering we talk about "user IDs" a lot I'm leaning towards "The badge ID."